index.html

```diff
@@ -33,11 +33,175 @@ <h2 id="id-my-name">Marc Raphael</h2>
+            </div>
+            <a href="#" target="_parent" class="class-project-link">
+              <span class="class-project-button">See this Project</span>
+              <img src="./images/Union.png" alt="union pic" class="class-union-pic">
+            </a>
+          </div>
+        </div>
+        
+        <!--project six-->
+        <div class="class-project">
+          <img src="./images/Rectangle21(6).png" alt="project image" class="class-project-image">
+          <div class="class-project-description">
+            <div class="class-project-languages">
+              <p class="class-project-name">Project name goes here</p>
+              <span class="class-project-span">HTML/CSS</span>
+              <span class="class-project-span">Ruby on rails</span>
+              <span class="class-project-span">Javascript</span>
+            </div>
+            <a href="#" target="_parent" class="class-project-link">
+              <span class="class-project-button">See this Project</span>
+              <img src="./images/Union.png" alt="union pic" class="class-union-pic">
+            </a>
+          </div>
+        </div>
+      </section>
+
+      <!--The section about my self-->
+      <section id="id-section-about-my-self">
+
+        <!--Introduction to the the project list-->
+        <h1 id="id-about-title">About</h1>
+
+        <!--litle description about me-->
+        <p id="id-about-description">
+          Lorem ipsum dolor sit amet, consectetur adipiscing elit, 
+          sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. 
+          Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ea eiusmod
+        </p>
+
+        <!--Button to the CV-->
+        <a href="#" target="_parent" id="id-about-cv">
+          <span id="id-about-span">Get my CV</span>
+          <img src="./images/ic_download.svg" alt="union pic" id="id-about-pic">
+        </a>
+
+        <!--Languages used-->
+        <div class="class-about-div" id="id-about-languages">
+          <h2 class="class-about-div-title">Languages</h2>
+          <ul class="class-about-div-ul">
+            <li class="class-div-list">JavaScript(ES5/ES6)</li>
+            <li class="class-div-list">Ruby</li>
+            <li class="class-div-list">HTML</li>
+            <li class="class-div-list">CSS</li>
+          </ul>
+        </div>
+
+        <!--Frameworks used-->
+        <div class="class-about-div" id="id-about-frameworks">
+          <h2 class="class-about-div-title">Frameworks</h2>
+          <ul class="class-about-div-ul">
+            <li class="class-div-list">Bootstrap</li>
+            <li class="class-div-list">Ruby on rails</li>
+            <li class="class-div-list">RSpec</li>
+            <li class="class-div-list">Capibara</li>
+            <li class="class-div-list">Selenium</li>
+          </ul>
+        </div>
+
+        <!--All skills-->
+        <div class="class-about-div" id="id-about-skills">
```

Comment:
- [ ] I see that you have grouped the list of technologies (e.g., "JavaScript", "HTML" and "CSS") under each category using `ul` -`li`, good job there! However, the categories (languages, Frameworks, Skills) themselves should be grouped using `ul` - `li` as well.

Reply:
- [ ] Good changes are made here but, you wrap the heading (category) itself along with its list components (eg: - You wrapped Language, HTML, CSS, ... all together,  but Language should be out of the list). See inline comment for the new under the new review,

Reply:
If for this reason my code is not validated, you will be held responsible for this. I have applied the changes previously requested, and now you are asking me to undo these changes to apply the previous ones. it is time unnecessarily wasted that you are making me take!

Reply:
Please have a decision now on what changes I may apply!